pyproject.toml

```diff
@@ -47,11 +47,12 @@ requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
 build-backend = "poetry_dynamic_versioning.backend"
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = ">=3.9,<3.11"
```

Comment:
maybe this fails some of the PR checks?
see [failing logs](https://qaz.myakove.com/webhook_server/python-module-install/openshift-python-wrapper-PR-1528-cf6f7379a3e602e66da204f8263568f10c966c5f) saying `ERROR: Package 'openshift-python-wrapper' requires a different Python: 3.11.6 not in '<3.11,>=3.9'`

Reply:
So is tox failing simply because `tox.ini` file doesn't specify a valid ('<3.11,>=3.9') Pyhton version?

Reply:
you cannot remove pythonh3.8 support from the project; find numpy release which does work with 3.8